api/features/models.py

```diff
@@ -947,6 +947,48 @@ def _is_more_recent_version(self, other: "FeatureState") -> bool:
             and self.version > other.version
         ) or (self.version is not None and other.version is None)
 
+    @staticmethod
+    def copy_identity_feature_states(
+        target_identity: "Identity", source_identity: "Identity"
+    ) -> None:
+        target_feature_states: dict[str, FeatureState] = (
+            target_identity.get_overridden_feature_states()
+        )
+        source_feature_states: dict[str, FeatureState] = (
+            source_identity.get_overridden_feature_states()
+        )
```

Comment:
I think these dicts have `int`s instead of `str`.

```suggestion
        target_feature_states: dict[int, FeatureState] = (
            target_identity.get_overridden_feature_states()
        )
        source_feature_states: dict[int, FeatureState] = (
            source_identity.get_overridden_feature_states()
        )
```

Reply:
Right. I will do this as part of applying Matt's suggestion